src/scripts/Providers/OSUI/RangeSlider/NoUISlider/Utils.ts

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace Providers.OSUI.RangeSlider.NoUISlider.Utils {
+	export function SetRangeValues(providerConfigs: ProviderConfigs): ProviderConfigs {
+		const rangeSliderConfigs = JSON.parse(providerConfigs);
```

Comment:
private const, so it should be _rangeSliderConfigs

Reply:
fixed